src/app/clusters/barrier-control-server/barrier-control-server-cli.c

```diff
@@ -37,6 +37,7 @@
  *******************************************************************************
  ******************************************************************************/
 
+#if 0
```

Comment:
better to remove this file than to comment it out.  we can resurrect it trivially when needed